front/app/components/PostShowComponents/Comments/InternalComments.tsx

```diff
@@ -0,0 +1,203 @@
+// libraries
```

Comment:
File not touched save for naming of component. I just moved comments here and put tabbed navigation at the index page